specs/mediawiki/v1/content.yaml

```diff
@@ -62,12 +62,45 @@ paths:
       x-backend-request:
         uri: /{domain}/sys/page_revisions/page/
 
+  /{module:page}/title/{title}:
+    get:
+      tags:
+        - Page content
+      description: >
+        Returns the latest revision for the title. By default returns the latest revision
```

Comment:
Not fond of stonewalling but I'd like that to be clear for users. They needn't know what RESTBase is, not where it gets its data, so something along the lines of "Returns the latest revision ID for the title present in storage. Supply the Cache-Control: no-cache header to obtain the latest available revision ID".

(Also note "revision ID", not just "revision")